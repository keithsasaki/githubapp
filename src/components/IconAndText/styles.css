.icon-and-text img{
    width: 2.5rem;
    height: 2.5rem;
}

.icon-and-text {
    padding-bottom: 1rem;
    font-size: 2.1rem;
    font-weight: 300;
    color: var(--color-gray)
}
