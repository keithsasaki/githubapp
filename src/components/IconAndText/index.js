import React from 'react';

import './styles.css';

function IconAndText(props) {
    return(
        <div className="icon-and-text">
            <img src={props.icon} alt={props.alt} />
            <span> {props.text} </span>
        </div>
    );
}

export default IconAndText;