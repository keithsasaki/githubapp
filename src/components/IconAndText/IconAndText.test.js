import React from 'react';
import '@testing-library/jest-dom';

import renderer from 'react-test-renderer';
import IconAndText from '.';

describe("Testing IconAndText Component", () => {
    it("matches snapshot", () => {
        const tree = renderer.create(<IconAndText />).toJSON();
        expect(tree).toMatchSnapshot();
    });
})
