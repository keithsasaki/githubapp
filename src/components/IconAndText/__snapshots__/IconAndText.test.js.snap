// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing IconAndText Component matches snapshot 1`] = `
<div
  className="icon-and-text"
>
  <img />
  <span>
     
     
  </span>
</div>
`;
